torch/nn/functional.py

```diff
@@ -1228,6 +1228,10 @@ def adaptive_avg_pool2d(input: Tensor, output_size: BroadcastingList2[int]) -> T
     """
     if has_torch_function_unary(input):
         return handle_torch_function(adaptive_avg_pool2d, (input,), input, output_size)
+    if not isinstance(output_size, int):
+        if len(output_size)!=2:
```

Comment:
nit: I think you might need to fix lint here
```suggestion
        if len(output_size) != 2:
```